front-end/src/components/league-page/leaderboard/leaderboardCard.tsx

```diff
@@ -0,0 +1,41 @@
+import { SxProps, Theme,  useTheme } from '@mui/material';
+import { Box } from '@mui/system';
+import { getGradient } from '../general';
+import { darken } from '@mui/system/colorManipulator';
+import React from 'react';
+import { LeaderboardRowProps, Row } from './row';
+
+
+interface LeaderboardCardProps {
+    sortedRowProps: LeaderboardRowProps[]
+    header: LeaderboardRowProps,
+    title: React.ReactNode,
+    sx?: SxProps<Theme> | undefined;
+    width: number,
+    height: number
+}
+
+export default function LeaderboardCard(props: LeaderboardCardProps) {
+  const theme = useTheme() as Theme;
+
+  return (
+    <Box sx={{
+      minWidth: props.width,
+      minHeight: props.height,
+      flexGrow: 1,
+      flexWrap: 'wrap',
+    }}>
+      <Box sx={{ p: 0.2, borderRadius: '4px 4px 0px 0px', background: getGradient(theme.palette.risenBoxBg.main)  }}>
```

Comment:
I know this is a decent sized refactor, but can basically all of the gradients be moved to the risenBox? And have a parameter like useGradient, so it would be easy to ship to all pages later if we want. This file in particular should then be able to use a RisenBox as the top level box with the gradient I think.

Reply:
For a lot of the boxes i use in this page the risen box doesnt actually meet the design i was going for so i ended up not using it for a lot of things. I can make a followup task to clean it up but id prefer not to do it in this CR since its already big and i dont really want add 40+ more files when i refactor

Reply:
[****](https://github.com/users/afieldi/projects/2/views/1?pane=issue&itemId=33971381)